src/vivarium_csu_ltbi/data/household_tb_model.py

```diff
@@ -14,15 +14,22 @@ def load_household_input_data(country: str):
     input_data_path = household_tb_paths.get_input_data_path(country)
 
     df = pd.read_stata(input_data_path)
-    df['hh_id'] = df['hh_id'].str.split().map(lambda x: int(''.join(x)))
-    df['sex'] = df['sex'].str.capitalize()
-    if formatted_country(country) == 'philippines':
-        df['age'] = df['age'].replace('96+', 95)
+
+    if formatted_country(country) in ['ethiopia', 'india', 'philippines']:
```

Comment:
Why are outputs different for different countries?

Reply:
Are you refer to their age formatted in microdata?

Reply:
Yep